test/unit/mdc-list/mdc-list.test.js

```diff
@@ -156,6 +156,12 @@ test('#initializeListType does not populate selectedIndex when no item is select
   td.verify(mockFoundation.setSelectedIndex(td.matchers.anything()), {times: 0});
 });
 
+test('#setEnabled when true adds disabled class and aria-disabled to true', () => {
+  const {component, mockFoundation} = setupTest();
+  component.setEnabled(1, true);
```

Comment:
did you mean false here? This list item is already enabled.

Reply:
You're right, I changed it to a better testing tittle.